### PR TITLE
Add PReLU API conversion interface.

### PR DESCRIPTION
**Pull request by fredtony**
Removed BatchNormalization and changed PReLU legacy support to fit new abstraction function.

### CODE REVIEW

File: tests/keras/legacy/interface_test.py [hunk 1]
```diff
@@ -59,5 +60,12 @@ def test_avgpooling1d_legacy_interface():
     assert json.dumps(old_layer.get_config()) == json.dumps(new_layer.get_config())
 
 
+@keras_test
+def test_prelu_legacy_interface():
+    old_layer = keras.layers.PReLU(init='zero', weights=zeros(5), name='p')
```
Comment (fchollet) [inline tests/keras/legacy/interface_test.py]: `weights` are part of the base layer kwargs, so it doesn't need to be included in the test. `name` is only included in order to get the test to pass (otherwise the different names would make the configs different). You can remove it, especially since this is not be the proper syntax for this argument (needs a list).

Comment (fredtony) [inline tests/keras/legacy/interface_test.py]: Yeah, I noticed they went back to the Layer class, but put it in for my own check to ensure it was still working. I'll take it out.  I'm keeping the `name` argument in since I saw that it failed when I took it out.

**Comment by fchollet** [comment]
LGTM, thanks!
